migrations/2023-02-05-154242_registration_review_content/up.sql

```diff
@@ -0,0 +1,12 @@
+-- add new registration mode
+alter type registration_mode_enum add value 'review_content';
+
+-- add table for comment review queue
+create table review_comment (
+  id            serial    primary key,
```

Comment:
Sorry to have to do this, but it should probably be named `comment_review`,  similar to `comment_report` . That'll also require name changes for the diesel table and views.